specification/dartLangSpec.tex

```diff
@@ -23406,40 +23410,15 @@ \subsubsection{Comments}
 Dart supports both single-line and multi-line comments.
 A \Index{single line comment} begins with the token \code{//}.
 Everything between \code{//} and the end of line
-must be ignored by the Dart compiler
-unless the comment is a documentation comment.
+must be ignored by the Dart compiler.
 
 \LMHash{}%
 A \Index{multi-line comment} begins with the token \code{/*}
 and ends with the token \code{*/}.
-Everything between \code{/}* and \code{*}/
-must be ignored by the Dart compiler
-unless the comment is a documentation comment.
+Everything between \code{/*} and \code{*/}
+must be ignored by the Dart compiler.
 Comments may nest.
```

Comment:
(That somewhat contradicts the "everything between `/*` and `*/` must be ignored. But not something to fix in this CL.)

Reply:
I had the same thought (both of them), but I don't think it's bad enough to look for a more precise wording. ;-)